### PR TITLE
fix: OPTIC-988: Convert styl files to scss

### PR DESCRIPTION
**Pull request by carly-bartel**
converting styl files to scss in LS web/libs/editor/src/common/

### CODE REVIEW

File: web/libs/editor/src/common/Button/Button.scss [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+      box-shadow: none;
+      box-shadow: inset 0px -1px 0px rgba(0, 0, 0, 0.1);
+
+      &:disabled {
+        --button-color: var(--sand_500);
+        background-color: var(--sand_200);
+      }
+
+      &:hover {
+        color: var(rgba(#fff, 0.8));
```
Comment (yyassi-heartex) [inline web/libs/editor/src/common/Button/Button.scss]:
```suggestion
        color: rgba(255, 255, 255, 0.8);
```

File: web/libs/editor/src/common/Button/Button.scss [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+        background: linear-gradient(0deg, rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0.1)), var(--primary_link);
+        box-shadow: 0px 2px 4px var(--grape_100), inset 0px -1px 0px rgba(0, 0, 0, 0.1);
+      }
+
+      &:active {
+        color: var(rgba(#fff, 0.8));
```
Comment (yyassi-heartex) [inline web/libs/editor/src/common/Button/Button.scss]:
```suggestion
        color: rgba(255, 255, 255, 0.8);
```

File: web/libs/editor/src/common/Tooltip/Tooltip.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        transform: translate(-50%, -50%) rotate(45deg);
+      }
+    }
+  }
+
+  &_theme {
+    &_light {
+      color: var(--sand_900);
+      background-color: var(--sand_0);
+      box-shadow: 0 2px 15px rgba(#000, 0.2);
```
Comment (yyassi-heartex) [inline web/libs/editor/src/common/Tooltip/Tooltip.scss]:
```suggestion
      box-shadow: 0 2px 15px rgba(0, 0, 0, 0.2);
```

**Review by yyassi-heartex — APPROVED** [review]

**Review by MihajloHoma — APPROVED** [review]

File: web/libs/editor/src/common/Dropdown/DropdownComponent.tsx
Comment (hlomzik) [inline web/libs/editor/src/common/Dropdown/DropdownComponent.tsx:1]: Dropdown.styl is not removed here?

File: web/libs/editor/src/common/Dropdown/Dropdown.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    opacity: 1;
+    display: flex;
+    transform: translate3d(0, 0, 0);
+  }
+
+  &.disappear {
+    opacity: 0;
+    transform: translate3d(0, var(--menu-animation-start), 0);
+  }
+}
```
Comment (hlomzik) [inline web/libs/editor/src/common/Dropdown/Dropdown.scss]: do you have stylelint running? I believe we had rules to have new line at the end

**Review by hlomzik — CHANGES_REQUESTED** [review]
tiny questions

File: web/libs/editor/src/common/Dropdown/DropdownComponent.tsx
Comment (yyassi-heartex) [reply web/libs/editor/src/common/Dropdown/DropdownComponent.tsx:1]: must have been missed by accident - thats ok any missed files i'll remove during the last ticket in the epic when I go through and remove the stylus libraries from LSO and LSE.
i'll go ahead and remove Dropdown.styl now though - great catch!

Comment (yyassi-heartex) [reply web/libs/editor/src/common/Dropdown/DropdownComponent.tsx:1]: file removed

File: web/libs/editor/src/common/Dropdown/Dropdown.scss (hunk 4 above)
Comment (yyassi-heartex) [reply web/libs/editor/src/common/Dropdown/Dropdown.scss]: looked into this briefly - it looks like the linter hasn't been merged yet - https://github.com/HumanSignal/label-studio/pull/6134 - i think that will happen after the stylus conversion is done to reduce merge conflict resolution

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10387089409)
> Successfully merged:  79 files changed, 2391 insertions(+), 1614 deletions(-)

File: web/libs/editor/src/common/Dropdown/Dropdown.scss (hunk 4 above)
Comment (yyassi-heartex) [inline web/libs/editor/src/common/Dropdown/Dropdown.scss:62]:
```suggestion
}

```

Comment (yyassi-heartex) [reply web/libs/editor/src/common/Dropdown/Dropdown.scss]:
```suggestion
}

```

File: web/libs/editor/src/common/Button/Button.scss [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+
+@keyframes button-waiting {
+  0% {
+    background-position: 0 0;
+  }
+
+  100% {
+    background-position: 37px 0;
+  }
+}
```
Comment (yyassi-heartex) [inline web/libs/editor/src/common/Button/Button.scss]:
```suggestion
}

```

**Review by hlomzik — APPROVED** [review]

**Pull request merged**